### PR TITLE
GUI: Check if fullscreen is available at runtime

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: backends/graphics/symbiansdl/symbiansdl-graphics.cpp [hunk 1]
```diff
@@ -33,7 +33,6 @@ SymbianSdlGraphicsManager::SymbianSdlGraphicsManager(SdlEventSource *sdlEventSou
 
 bool SymbianSdlGraphicsManager::hasFeature(OSystem::Feature f) const {
 	switch (f) {
-	case OSystem::kFeatureFullscreenMode:
```
Comment (bluegr) [inline backends/graphics/symbiansdl/symbiansdl-graphics.cpp]: Why did you remove this?

Comment (ccawley2011) [reply backends/graphics/symbiansdl/symbiansdl-graphics.cpp]: On Symbian, the "fullscreen" config option is ignored, and the fullscreen checkbox in the GUI is disabled using `GUI_ONLY_FULLSCREEN`, which suggests that Symbian doesn't actually support fullscreen mode.

Comment (bluegr) [reply backends/graphics/symbiansdl/symbiansdl-graphics.cpp]: Indeed! Thanks for the explanation

**Comment by bluegr** [comment]
Very nice! Thanks for your work. Merging
